locations/spiders/morrisons_gb.py

```diff
@@ -0,0 +1,107 @@
+import string
+
+from scrapy import Spider
+
+from locations.categories import Categories
+from locations.dict_parser import DictParser
+from locations.hours import DAYS_EN, OpeningHours
+
+
+class MorrisonsSpider(Spider):
+    name = "morrisons_gb"
+    item_attributes = {"nsi_id": -1}  # Skip NSI matching as it
+    # conflates stores and
+    # fuel stations.
```

Comment:
I'm going to try and find the reason for this

Reply:
Thank you. I don't know this brand and it is confusing which location is a fuel station or convenience store, or both. It's clear that Morrisons (parent brand) has rebadged themselves quite a few times and this is constantly occurring.

Reply:
Even though Morrisons API gives [Select](https://www.google.com/maps/@52.3930928,-1.96802,3a,75y,141.46h,87.23t/data=!3m6!1e1!3m4!1s1RkC8TA6ZEVtXVreoRzZAg!2e0!7i16384!8i8192?entry=ttu) and [Daily](https://www.google.com/maps/@51.4950076,-2.6772714,3a,75y,144.15h,88.16t/data=!3m6!1e1!3m4!1sgVHWcS7NhaD50mHPjaE-rQ!2e0!7i16384!8i8192?entry=ttu) `category=Gas Station`, that's not actually the category. They are convenience stores, sometimes at petrol stations.

Yeah, local knowledge is super important in OSM, and also here. Morrisons is relatively stable, but they tried (and failed) with Morrisons Select. They then started Morrisons Daily, which was doing a little better, but not great. So they've bought McColl's and started rebranding them into Morrisons Daily. McColls also owns "Martin's" and "RS McColl", but I'm not sure why they still exist.